components/org.wso2.carbon.identity.local.auth.emailotp/src/main/java/org/wso2/carbon/identity/local/auth/emailotp/constant/AuthenticatorConstants.java

```diff
@@ -98,6 +98,7 @@ public static class ConnectorConfig {
         public static final String OTP_EXPIRY_TIME = "EmailOTP.ExpiryTime";
         public static final String ENABLE_BACKUP_CODES = "EmailOTP.EnableBackupCodes";
         public static final String EMAIL_OTP_LENGTH = "EmailOTP.OTPLength";
+        public static final String EMAIL_OTP_USE_ALPHANUMERIC_CHARS = "EmailOTP.OtpRegex.UseAlphanumericChars";
```

Comment:
`OtpRegex` feels redundant. is this something we are introducing or has it being already used?

Reply:
This is a newly introduced config.